docs: combine How it works and Setup sections in Prequel data sync page (#906)
knocklabs/docs: content/integrations/extensions/data-sync.mdx

## How it works

> Knock integrates with your data warehouse using [Prequel](https://www.prequel.co/). There is no self-service UI to connect your account – please [contact our support team to get started](mailto:support@knock.app). We will send you a secure link to provide information about your data warehouse, including its destination type. Depending on the destination type (supported databases listed [here](https://docs.prequel.co/docs/destinations-overview)), we may ask for additional details.

Reviewer: ```suggestion
Knock integrates with your data warehouse using [Prequel](https://www.prequel.co/). There is no self-service UI to connect your account – please [contact our support team to get started](mailto:support@knock.app). We will need some information about your data warehouse, including its destination type. Depending on the type (supported databases listed [here](https://docs.prequel.co/docs/destinations-overview)), we may ask for additional details.
```

> Once the necessary data warehouse information is provided, Knock will provide you with a unique magic link. This link will guide you through a step-by-step wizard tailored to your specific data warehouse destination. Upon successful connection and saving, a [backfill of historical data](https://docs.prequel.co/docs/transfer-logic#backfills--full-refreshes) will begin and you will continue to receive the most up-to-date data every 24 hours.

Reviewer: ```suggestion
Once the necessary data warehouse information is provided, Knock will provide you with a unique magic link. This link will guide you through a step-by-step wizard tailored to your specific data warehouse destination. Upon successful connection, a [backfill of historical data](https://docs.prequel.co/docs/transfer-logic#backfills--full-refreshes) will begin and you will continue to receive the most up-to-date data every 24 hours.
```
This one is more just a nit to make it more concise
